### DISCUSSION TITLE
Phishing mail from legit looking github mail address

### DISCUSSION THREAD

**Discussion by lauhon** [community/community#109433] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Show & Tell

### Body

I recieved two phishing mails today that are somewhat "well done" i.e. relatively easy to fall for.

The scammers managed to forge the from email address to be `notifications@github.com`. It seems they abused the email mechanism that's in place when you get notification from Issues you subscribed from.

In the mail is a Link that's probably malicious (haven't clicked it) to [https://auth.githubtalentcommunity.online/](DONT VISIT)

![image](https://github.com/community/community/assets/46261860/af077e54-bc8e-4b56-b37c-f33e769f26ef)

I wasn't sure where to report this, as I don't think its the Issues authors fault, nor do I have permission to report the issue in the first place. Hope this helps somehow

Also I want to note that I do not recall subscribing to those Issues at all - that's a bit scary to me, because it seems like someone managed to subscribe to the issue in my name...

**Comment by kleguizamon** [comment]
I received the same email from notifications@github.com, **it's a scam**

![image](https://github.com/community/community/assets/7070068/5c6f8038-954e-4c32-aaeb-791b38fd4785)

### More details:

https://x.com/realScamSniffer/status/1760124587298488508?s=20

**Comment by negue** [comment]
Received 3 of them today (6hrs ago) as well - seems a bot just picks a random PR fill in all the data with that fake auth link and mentions a few people - then after the notification is sent deletes the comment itself.

I really hope no body falls for this 🙏

**Reply by negue** [reply, marked as answer]
Seems there is already a discussion about this https://github.com/orgs/community/discussions/109171

**Reply by Bughue** [reply]
hello , this page is 404, and I have visited that link in email .now my account has been flagged.Do you know how I can resolved that?

**Comment by kunthar** [comment]
Another one:
https://github.com/refcell/adms/pull/31#issuecomment-1955867942

**Comment by ettaboyle** [comment]
We understand the inconvenience caused by these notifications. Our teams are currently working on addressing these unsolicited phishing notifications. We want to remind our users to continue to use [our abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam) to raise any abusive or suspicious activity.

We would like to bring to our users’ attention to the following:
- Please do not click any links or reply to these notifications. Please [report them](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam).
- [Authorizing an OAuth app](https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/authorizing-oauth-apps) can expose your GitHub account and data to a third party.  
- GitHub’s recruitment process would never mention its users via issues/PRs or other public content.
- GitHub recommends you periodically [review your authorized OAuth apps](https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/reviewing-your-authorized-oauth-apps). 
This spam activity targets and abuses GitHub’s mention and notification functionality.
- This is a phishing campaign and is not the result of a compromise of GitHub or its systems.

**Comment by vadim-kravtsov** [comment]
Received one just couple of hours ago, looked pretty legit.

<img width="618" alt="Screenshot 2025-03-15 at 20 51 13" src="https://github.com/user-attachments/assets/3d165325-b595-41d4-aa33-0e987daf68e2" />
